README.md

```diff
@@ -4,226 +4,31 @@ Airbyte Python CDK is a framework for building Airbyte API Source Connectors. It
 classes and helpers that make it easy to build a connector against an HTTP API (REST, GraphQL, etc),
 or a generic Python source connector.
 
-## Usage
+## Building Connectors with the CDK
 
-If you're looking to build a connector, we highly recommend that you
+If you're looking to build a connector, we highly recommend that you first
 [start with the Connector Builder](https://docs.airbyte.com/connector-development/connector-builder-ui/overview).
 It should be enough for 90% connectors out there. For more flexible and complex connectors, use the
 [low-code CDK and `SourceDeclarativeManifest`](https://docs.airbyte.com/connector-development/config-based/low-code-cdk-overview).
 
-If that doesn't work, then consider building on top of the
-[lower-level Python CDK itself](https://docs.airbyte.com/connector-development/cdk-python/).
-
-### Quick Start
-
-To get started on a Python CDK based connector or a low-code connector, you can generate a connector
-project from a template:
-
-```bash
-# from the repo root
-cd airbyte-integrations/connector-templates/generator
-./generate.sh
-```
-
-### Example Connectors
-
-**HTTP Connectors**:
-
-- [Stripe](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-stripe/)
-- [Salesforce](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-salesforce/)
-
-**Python connectors using the bare-bones `Source` abstraction**:
-
-- [Google Sheets](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-sheets/google_sheets_source/google_sheets_source.py)
-
-This will generate a project with a type and a name of your choice and put it in
-`airbyte-integrations/connectors`. Open the directory with your connector in an editor and follow
-the `TODO` items.
+For more information on building connectors, please see the [Connector Development](https://docs.airbyte.com/connector-development/) guide on [docs.airbyte.com](https://docs.airbyte.com).
```

Comment:
Honestly, that's the way. Those always get out of date, so it's good to just link to the general guide.